app/assess/models/round.py

```diff
@@ -30,10 +30,6 @@ def from_dict(cls, d: dict):
 
     @staticmethod
     def from_json(data: dict):
-        # eligibility_criteria = {}
-        # if "eligibility_criteria" in data:
-        #     for key, value in data["eligibility_criteria"].items():
-        #         eligibility_criteria.update({key: value})
```

Comment:
Can we remove this rather than commenting out?

Reply:
that has been removed, it was commented out originally but Im not sure why

Reply:
@Adam-W1 change doesn't seem to have been pushed to remove this?

Reply:
The red highlight means it's being removed (Im removing it from main as part of this BAU)